tests/cases/standalone/common/flow/flow_tql.result

```diff
@@ -0,0 +1,83 @@
+CREATE TABLE http_requests (
+  ts timestamp(3) time index,
+  host STRING,
+  idc STRING,
+  val BIGINT,
+  PRIMARY KEY(host, idc),
+);
+
+Affected Rows: 0
+
+CREATE FLOW calc_reqs SINK TO cnt_reqs AS
+TQL EVAL (now() - '1m'::interval, now(), '5s') count_values("status_code", http_requests);
+
+Affected Rows: 0
+
+SHOW CREATE TABLE cnt_reqs;
+
++----------+-------------------------------------------+
+| Table    | Create Table                              |
++----------+-------------------------------------------+
+| cnt_reqs | CREATE TABLE IF NOT EXISTS "cnt_reqs" (   |
+|          |   "count(http_requests.val)" BIGINT NULL, |
+|          |   "ts" TIMESTAMP(3) NOT NULL,             |
+|          |   "status_code" BIGINT NULL,              |
+|          |   "update_at" TIMESTAMP(3) NULL,          |
+|          |   TIME INDEX ("ts"),                      |
+|          |   PRIMARY KEY ("status_code")             |
+|          | )                                         |
+|          |                                           |
+|          | ENGINE=mito                               |
+|          |                                           |
++----------+-------------------------------------------+
+
+INSERT INTO TABLE http_requests VALUES
+    (now() - '17s'::interval, 'host1', 'idc1', 200),
```

Comment:
I think we can use fixed time to assert the results.

Reply:
> I think we can use fixed time to assert the results.

need to test if `now()` is working

Reply:
They are both needed.